### ISSUE TITLE
RX ANT на Панели E-Coder.

### ISSUE THREAD

**Issue by UA6AA** [ExpertSDR3/ExpertSDR3-SUGGESTIONS#171]
Прошу добавить возможность назначения на одну из кнопок E-Coder функции включения RX ANT.
Она куда-то пропала в eesdr3 версии.
В предыдущей она была.
Очень неудобно это делать сейчас мышкой.

**Comment by HA0LC** [comment]
I agree with him. It is annoying to use mouse in every cases we want to switch RX ANT on and off.
Please add this to MAIN menu for E-CODER. Tnx, Joe/HA0LC

**Comment by UD2F** [comment]
Поддерживаю. Нужна такая кнопка.
